pandas/tests/resample/test_timedelta.py

```diff
@@ -150,3 +150,15 @@ def test_resample_timedelta_edge_case(start, end, freq, resample_freq):
     tm.assert_index_equal(result.index, expected_index)
     assert result.index.freq == expected_index.freq
     assert not np.isnan(result[-1])
+
+
+def test_resample_timedelta_no_empty_groups():
```

Comment:
nit-pick: `test_resample_with_timedelta_yields_no_empty_groups`

Just to make it easier to tell what the test does if we see it failing